src/services/accounts/AccountsBalanceInfoService.ts

```diff
@@ -161,14 +174,89 @@ export class AccountsBalanceInfoService extends AbstractService {
 				at,
 				nonce,
 				tokenSymbol: token,
-				free,
-				reserved,
-				miscFrozen,
-				feeFrozen,
-				locks,
+				free: this.inDenominationBal(denominate, free, decimal),
+				reserved: this.inDenominationBal(denominate, reserved, decimal),
+				miscFrozen: this.inDenominationBal(denominate, miscFrozen, decimal),
+				feeFrozen: this.inDenominationBal(denominate, feeFrozen, decimal),
+				locks: this.inDenominationLocks(denominate, locks, decimal),
 			};
 		} else {
 			throw new BadRequest('Account not found');
 		}
 	}
+
+	/**
+	 * Apply a denomination to a balance depending on the chains decimal value.
```

Comment:
so I don't really understand why we can't just do `balance.toString() || "." || dec.toString()`?